apps/desktop/electron/database/tables/BeatTable.ts

```diff
@@ -85,11 +85,15 @@ export function getBeat({
     db: Database.Database;
     beatId: number;
 }): DbActions.DatabaseResponse<DatabaseBeat | undefined> {
-    return DbActions.getItem<DatabaseBeat>({
-        tableName: Constants.BeatsTableName,
-        db,
-        id: beatId,
-    });
+    const beat = getOrm(db)
```

Comment:
Dealer's choice between this and
```
    const beat = getOrm(db)
        .select()
        .from(schema.beats)
        .where(eq(schema.beats.id, beatId))
        .get();
```
The [relational query builder ](https://orm.drizzle.team/docs/rqb#querying)lets you do some cool stuff with joins that might be more useful in other tables.